ni/github.py

```diff
@@ -34,6 +34,10 @@
 
 {body}
 
+When your account is ready, please add a comment in this pull request
+and a Python core developer will remove the `CLA not signed` label
+to make the bot check again.
```

Comment:
Maybe this should be two sentences.

Is «Python core developer» accurate?

Reply:
No one else has the ability to change labels, so it's accurate.

Reply:
The question was more in the sense of «core dev» / «core reviewer» / «team member», but if the old term still works, good!

(I guess «Python committer» fell out of usage)

Reply:
Both terms work and I personally don't have a preference.